code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -45,6 +45,387 @@
+	and it’s said that blood runs down the blade in just the right way, to drip artfully from the twin ‘fangs’ at its apex."
+
+// Unathi Sword
+/obj/item/melee/breach_cleaver
+	name = "breach cleaver"
+	desc = "Massive, heavy, and utterly impractical. This sharpened chunk of steel is too big and too heavy to be called a sword."
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
+	base_icon_state = "breach_cleaver"
+	icon_state = "breach_cleaver0"
+	flags = CONDUCT
+	force = 5
+	throwforce = 5
+	w_class = WEIGHT_CLASS_BULKY
+	sharp = TRUE
+	origin_tech = "combat=6;syndicate=5"
+	attack_verb = list("slashed", "cleaved", "chopped")
+	hitsound = 'sound/weapons/swordhitheavy.ogg'
+	materials = list(MAT_METAL = 2000)
+	needs_permit = TRUE
+	///How much damage the sword does when wielded
+	var/force_wield = 40
+
+/obj/item/melee/breach_cleaver/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_wielded = force_wield, force_unwielded = force, icon_wielded = "[base_icon_state]1", wield_callback = CALLBACK(src, PROC_REF(wield)), unwield_callback = CALLBACK(src, PROC_REF(unwield)))
+	AddComponent(/datum/component/parry, _stamina_constant = 2, _stamina_coefficient = 0.5, _parryable_attack_types = NON_PROJECTILE_ATTACKS)
+
+/obj/item/melee/breach_cleaver/examine(mob/user)
+	. = ..()
+	if(isAntag(user))
+		. += "<span>When wielded, this blade has different effects depending on your intent, similar to a martial art. \
+			Help intent will strike with the flat, dealing stamina, disarm intent forces them away, grab intent knocks down the target, \
+			and harm intent deals heavy damage.</span>"
+
+/obj/item/melee/breach_cleaver/examine_more(mob/user)
+	. = ..()
+	. += "Massive, heavy, and utterly impractical. This sharpened chunk of steel is too big and too heavy to be called a sword."
+	. += ""
+	. += "The Unathi Breach Cleaver is a weapon the scaled, warlike race favours for its impressive weight and myriad combat applications. \
+	The pinnacle of Moghes' combat technology, it combines all of this knowledge into a massive, heavy slab of alloyed metal that most \
+	species find difficult to lift, let alone use in any sort of fight."
+	. += ""
+	. += "Actually a little lightweight for its size, a Breach Cleaver is unmatched in combat utility as a weapon, a tool for getting into\
+	places and as a slab of armour for the wielder. The leather of the Kar'oche beast, a predator native to Moghes, binds the hilt, \
+	allowing it to be gripped securely by its warrior. The wide blade is often etched with scenes depicting military victories or great hunts."
+
+/obj/item/melee/breach_cleaver/update_icon_state()
+	icon_state = "[base_icon_state]0"
+
+/obj/item/melee/breach_cleaver/proc/wield(obj/item/source, mob/living/carbon/human/user)
+	to_chat(user, "<span class='notice'>You heave [src] up in both hands.</span>")
+	user.apply_status_effect(STATUS_EFFECT_BREACH_AND_CLEAVE)
```

Comment:
does this status effect get cleared off the user in most use cases?
1. User drops the sword
2. Sword gets QDEL'd
3. Sword is teleported out of users hand

Reply:
Tested it, status effect is removed if:
- Disarmed/Dropped
- Deleted
- Thrown

Reply:
good work, thank you